src/utils/statisticsCalculations.ts

```diff
@@ -39,20 +40,29 @@ async function getTotalTransactionsChartData(groupBy) {
   return { timeStamps, overallValues }
 }
 
+async function saveStatistic(
+  totalBlocks: number,
+  totalTransactions: number,
+  totalWalletAddresses: number,
+  totalAssets: number,
+  totalTransactionsChartData: any
+) {
+  Statistic.create({ totalBlocks, totalTransactions, totalWalletAddresses, totalAssets, totalTransactionsChartData })
+}
+
```

Comment:
suggestion(non-blocking): I suggest using an object here as more than 3 parameters is always a little messy in my opinion. You can create a new data type like `stats` so that you benefit from the type checking.